TYPO3.Neos/Resources/Public/JavaScript/Content/Inspector/Editors/ImageEditor.js

```diff
@@ -298,9 +298,18 @@ function (Ember, $, FileUpload, template, cropTemplate, BooleanEditor, Spinner,
 		_mediaBrowserView: null,
 
 		_beforeMediaBrowserIsShown: function () {
-			var that = this;
+			var that = this,
+				value = null;
+
+			try {
+				value = JSON.parse(this.get('value'));
+			} catch (exception) {}
+
 			window.Typo3MediaBrowserCallbacks = {
-				assetChosen: function (assetIdentifier) {
+				_assetIdentifier: value && '__identity' in value ? value.__identity : null,
+				_reloadPreviewImage: function(assetIdentifier) {
```

Comment:
the value doesn't need to be parsed does it? it's available as `this._assetIdentifier` right?